pkg/core/core.go

```diff
@@ -130,17 +133,36 @@ func (comic *Comic) makePDF(options *config.Options) error {
 		return err
 	}
 
+	imageOptions := gofpdf.ImageOptions{ImageType: util.ImageType(comic.ImagesFormat), ReadDpi: true, AllowNegativePosition: false}
 	for _, file := range files {
-		pdf.AddPage()
-		imageOptions := gofpdf.ImageOptions{ImageType: util.ImageType(comic.ImagesFormat), ReadDpi: true, AllowNegativePosition: false}
-		fileName := fmt.Sprintf("%s/%s", imagesPath, file.Name())
+		mmWd = 210.0
+		mmHt = 297.0
+		fileName := fmt.Sprintf("%s%s%s", imagesPath, string(os.PathSeparator), file.Name())
+
+		if !options.ForceAspect {
```

Comment:
Something like that could work? In this way we avoid the last `else` statetement
```go

if !options.ForceAspect {
	img, err := os.Open(fileName)

	defer img.Close() // (?)

	if err != nil {
		options.Logger.Error(err.Error())

	}
	im, _, err := image.DecodeConfig(img)
	if err != nil {
		options.Logger.Error(err.Error())
	} else {
		mmWd = px2mm * float64(im.Width)
		mmHt = px2mm * float64(im.Height)
	}
} 
```

Reply:
Yeah, I think that will do it. I don't know golang well enough to know when img.Close() will happen, so wanted to make sure it was closed before the file is again accessed with ioutil.ReadFile(). On WIndows, I'm worried that both opens simultaneously might give an access error, but I don't know for sure.

I can test in a few hours, the site antibots are on to me right now.

Reply:
> Whops, a little more - I had the nested if statements because I only want to decode the image if the open was valid. In the code above, the image decode will be attempted regardless. If that's OK, then sure, this can work? Again, I've been writing golang for like 24 hours now 🤣

The `os.Open` should be fine if the file exist 😄